package-lock.json

```diff
@@ -420,6 +488,36 @@
         "node": ">=20.0.0"
       }
     },
+    "eng/tools/summarize-impact": {
+      "name": "@azure-tools/summarize-impact",
+      "dev": true,
+      "dependencies": {
+        "@azure-tools/openapi-tools-common": "^1.2.2",
+        "@azure-tools/specs-shared": "file:../../../.github/shared",
+        "@azure/openapi-markdown": "0.9.4",
+        "@ts-common/commonmark-to-markdown": "^2.0.2",
+        "commonmark": "^0.29.0",
+        "glob": "^11.0.3",
```

Comment:
Maybe in an future PR, but could this be switch to `globby@^14`, which we already use in TSV?